views/app-landing.hbs

```diff
@@ -0,0 +1,41 @@
+<h2 id="compliment">{{randComp1.compliment}}</h2>
+
+<!-- <h3>{{comps.length}} compliments</h3>
+
+{{#each comps as |comp index|}}
+  <h4>{{comp.id}}. {{comp.compliment}}</h4>
+{{/each}} -->
+
+<button type="button" name="button" class="btn btn-primary" id="new-comp">New Compliment</button>
+
+<script type="text/javascript">
+var button = $("#new-comp");
+
+button.on("click", changeCompliment);
+
+var isOne = true;
+
+function changeCompliment(event){
+  event.preventDefault();
+  if (isOne == true) {
+    $("#compliment").html("{{randComp2.compliment}}");
+    changeColor();
+    isOne = false;
+  } else if (isOne == false) {
```

Comment:
You only need to do this once either server side or client side, which might make it more simple. Call `Math.floor(Math.random())` server side, then just render the hbs view using `{{compliment}}`